src/common/xo/index.js

```diff
@@ -1875,6 +1877,24 @@ export const deleteSshKey = key =>
     })
   }, noop)
 
+export const deleteSshKeys = keys =>
+  confirm({
+    title: _('deleteSshKeysConfirm', { nKeys: keys.length }),
+    body: _('deleteSshKeysConfirmMessage', {
+      nKeys: keys.length,
+    }),
+  }).then(() => {
+    const { preferences } = xo.user
+    map(keys, key =>
+      _setUserPreferences({
+        sshKeys: filter(
+          preferences && preferences.sshKeys,
+          k => !isEqual(k, key)
```

Comment:
Since the `keys` objects passed to `deleteSshKeys` are actually augmented with the `id` property, they will never be equal (within the meaning of `_.isEqual`), right?
So you should compare the actual key strings instead.

Reply:
yes that's it